server/app/mutations/grid_services/common.rb

```diff
@@ -84,6 +84,26 @@ def build_grid_service_secrets(existing_secrets)
       service_secrets
     end
 
+    def build_service_volumes()
+      service_volumes = []
+      self.volumes.each do |vol|
+        # TODO Check existing volumes, if needed
+        service_volume = nil
+        elements = vol.split(':')
+        if elements[0].start_with?('/') && elements[1] && elements[1].start_with?('/') # Bind mount
+          service_volume = ServiceVolume.new(bind_mount: elements[0], path: elements[1], flags: elements[2..-1].join(':'))
+        elsif !elements[0].start_with?('/') && elements[1].start_with?('/') # Real volume
+          volume = self.grid.volumes.find_by(name: elements[0])
+          # TODO What if the volume is not found?
+          service_volume = ServiceVolume.new(volume: volume, path: elements[1], flags: elements[2..-1].join(':'))
+        elsif elements[0].start_with?('/') && (elements.size == 1 || !elements[1].start_with?('/')) # anon volume
+          service_volume = ServiceVolume.new(bind_mount: nil, path: elements[0], flags: nil) # anon vols do not support flags
```

Comment:
I guess I was preparing for possibilitythat anon vol might have flags also. but that should not be possible afaik.